Formula/spr.rb

```diff
@@ -0,0 +1,23 @@
+class Spr < Formula
+  desc "Submit pull requests for individual, amendable, rebaseable commits to GitHub"
+  homepage "https://github.com/getcord/spr"
+  url "https://github.com/getcord/spr.git",
+    tag:      "v1.3.3",
+    revision: "1f6d99ef268643cead1341cfc51e57b84c50b962"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "zlib"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "spr #{version}", shell_output("#{bin}/spr --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
The difficulty is that you need GitHub credentials to do anything with this command line tool. Calling it with `--version` or `--help` are the only non-interactive operations you can do that does not require you to feed it a GitHub access token.

Reply:
Well, we can also do something that requires a GitHub token without actually providing one and then check that we get the expected error (exit code + error message).